Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -134,6 +134,7 @@ final class NetworkProtectionConnectionTester {
     func stop() async {
         os_log("🔴 Stopping connection tester", log: log)
         await stopScheduledTimer()
+        isRunning = false
```

Comment:
This should not be about the internal timer.  It should instead be a flag to signal the monitor in its entirety is running.

For this reason, this flag should be set to `false` only on the public `stop()` interface.

Reply:
This makes sense, nice one.